### ISSUE TITLE
ToDos' Supervised Learning

### ISSUE THREAD

**Issue by juan190199** [juan190199/ML-From-Scratch#2]
- Adaboost
- Bayesian Regression
- Decision Tree
- Gradient Boosting
- Multilayer Perceptron
- Neuroevolution
- Particle Swarm Optimization of Neural Network
- Perceptron
- Random Forest
- Support Vector Machine
- XGBoost

**Comment by juan190199** [comment]
- Adaboost
- Bayesian Regression
- Gradient Boosting
- Multilayer Perceptron
- Neuroevolution
- Particle Swarm Optimization of Neural Network
- Perceptron
- Random Forest
- Support Vector Machine
- XGBoost

**Comment by juan190199** [comment]
- Adaboost
- Bayesian Regression
- Gradient Boosting
- Neuroevolution
- Particle Swarm Optimization of Neural Network
- Random Forest
- Support Vector Machine
- XGBoost

**Comment by juan190199** [comment]
- Adaboost
- Bayesian Regression
- Generalised Linear Models
- Gradient Boosting
- Neuroevolution
- Particle Swarm Optimization of Neural Network
- Random Forest
- Support Vector Machine
- XGBoost
